ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -138,6 +147,127 @@ export const AccountListMenu = ({ onClose }) => {
             />
           </Box>
         ) : null}
+        {/* Add / Import / Hardware Menu */}
+        {actionMode === 'menu' ? (
+          <Box padding={4}>
+            <Box>
+              <ButtonLink
+                size={Size.SM}
+                startIconName={IconName.Add}
+                onClick={() => {
+                  trackEvent({
+                    category: MetaMetricsEventCategory.Navigation,
+                    event: MetaMetricsEventName.AccountAddSelected,
+                    properties: {
+                      account_type: MetaMetricsEventAccountType.Default,
+                      location: 'Main Menu',
+                    },
+                  });
+                  setActionMode('add');
+                }}
+                data-testid="multichain-account-menu-popover-add-account"
+              >
+                {t('addNewAccount')}
+              </ButtonLink>
+            </Box>
+            <Box marginTop={4}>
+              <ButtonLink
+                size={Size.SM}
+                startIconName={IconName.Import}
+                onClick={() => {
+                  trackEvent({
+                    category: MetaMetricsEventCategory.Navigation,
+                    event: MetaMetricsEventName.AccountAddSelected,
+                    properties: {
+                      account_type: MetaMetricsEventAccountType.Imported,
+                      location: 'Main Menu',
+                    },
+                  });
+                  setActionMode('import');
```

Comment:
Optional: can we have local const to store the type of actionMode including `add`, `import` and `menu`. so there won't be any typo error when we use them :)

Reply:
Great idea!  Updated!